core/src/main/java/com/alibaba/nacos/core/code/ControllerMethodsCache.java

```diff
@@ -101,49 +163,77 @@ public void initClassMethod(Class<?> clazz) {
                     requestMethods[0] = RequestMethod.GET;
                 }
                 for (String methodPath : requestMapping.value()) {
-                    methods.put(requestMethods[0].name() + "-->" + classPath + methodPath, method);
+                    String urlKey = requestMethods[0].name() + REQUEST_PATH_SEPARATOR + classPath
+                            + methodPath;
+                    addUrlAndMethodRelation(urlKey, requestMapping.params(), method);
                 }
             }
         }
     }
-    
+
     private void parseSubAnnotations(Method method, String classPath) {
-        
+
         final GetMapping getMapping = method.getAnnotation(GetMapping.class);
         final PostMapping postMapping = method.getAnnotation(PostMapping.class);
         final PutMapping putMapping = method.getAnnotation(PutMapping.class);
         final DeleteMapping deleteMapping = method.getAnnotation(DeleteMapping.class);
         final PatchMapping patchMapping = method.getAnnotation(PatchMapping.class);
-        
+
```

Comment:
Sorry, do not change any indent from original
Please use nacos code style to reformat code .

detail see src/style/codeStyle.md

Reply:
ye.I set the nacos-code-style then format